README.md

```diff
@@ -44,6 +44,12 @@ make release
 make install
 ```
 
+### On Neon Postgres
+To quickly install the **pg_mooncake** extension on Neon, [create a Neon project](https://console.neon.tech/signup) and run the following commands from the [Neon SQL Editor](https://neon.tech/docs/get-started-with-neon/query-with-neon-sql-editor) or a [connected SQL client such as psql](https://neon.tech/docs/connect/query-with-psql-editor):
+```sql
+SET neon.allow_unstable_extensions='true';
+CREATE EXTENSION pg_mooncake;
```

Comment:
rm `CREATE EXTENSION pg_mooncake;`
That part is shared and documented in the next section

Reply:
thank you. resolved.